### PR TITLE
[WIP] Add Ember

### PR DESCRIPTION
**Pull request by locks**
### TODO
- [ ] Review idioms
- [ ] "no prop types"

### CODE REVIEW

**Comment by jeffcarp** [comment]
Beautiful, thank you for taking this up so quickly! Let me know if I can help.

**Comment by locks** [comment]
No problem :) I need to confirm a couple things, and I need to add a "no prop types" comment. There is an addon that adds them, should I add a remark saying as much?

File: _data/frameworks.yml [hunk 1]
```diff
@@ -108,6 +119,7 @@
       angular1: <div ng-repeat="item in items"></div>
       polymer: <template is="dom-repeat" items="{{items}}"></template>
       vue: <div v-for="item in items"></div>
+      ember: {{#each items as |item}}<div></div>{{/div}}
```
Comment (rinti) [inline _data/frameworks.yml]: missing the ending pipe

**Comment by jeffcarp** [comment]
I can't find any usage statistics but I'm guessing Ember is more popular than Polymer and Vue. I realize this means a lot of painful refactoring but do you want to move Ember up so that the order goes: React, Angular(s), Ember, Polymer, Vue?

I can totally do this after your PR lands. Or I can make the order less painful to change by only defining it once.

**Comment by jeffcarp** [comment]
Hi @locks, I decided to merge this as 2f8474165b511fdddec5a97bc891eccc0e7594e7 since it's mostly complete. Feel free to add anything else in a followup PR!

**Comment by locks** [comment]
Cheers :)

**Comment by jeffcarp** [comment]
Thanks for your work!!

**Comment by locks** [comment]
2.6 is releasing this week. Should I send a PR to update the version number?

**Comment by jeffcarp** [comment]
Not unless there's a change to the code

**Pull request closed without merging**